src/components/header/QuayHeader.tsx

```diff
@@ -14,12 +14,25 @@ import BarsIcon from '@patternfly/react-icons/dist/js/icons/bars-icon';
 import logo from 'src/assets/RH_QuayIO2.svg';
 import {HeaderToolbar} from './HeaderToolbar';
 import {Link} from 'react-router-dom';
+import {SidebarState} from 'src/atoms/SidebarState';
+import {useRecoilState} from 'recoil';
 
 export function QuayHeader() {
+  const [_sidebarState, setSidebarState] = useRecoilState(SidebarState);
+
+  const toggleSidebarVisibility = () => {
+    console.log('toggleSidebarVisibility', _sidebarState.isOpen);
```

Comment:
nit: could we remove the `console.log`

Reply:
oops should have forgot to remove it. Fixed